frontend/src/components/SubscribeMail/SubscribeMail.styles.sass

```diff
@@ -0,0 +1,64 @@
+.subscription-form
+  display: flex
+  flex-direction: column
+  align-items: center
+  margin-bottom: 5% 
+  text-align: center
+
+  label
+    font-size: 25px
+  
+  .msg
+    display: block
+    margin-top: 20px
+    font-size: 25px
+    margin-bottom: 0
+    &.success
+      color: #3cb878d9
+    &.error
+      color: red
+    &.success::after
+      content: 'Köszönjük feliratkozását!'
+    &.error::after
+      content: 'Valahol hiba lépett fel a feliratkozás során! :('
```

Comment:
@tamas-w this should be in the component instead of CSS
CSS is for styling not for content